.circleci/config.yml

```diff
@@ -83,11 +83,46 @@ jobs:
           --source client-java@$CIRCLE_SHA1 \
           --targets docs:development examples:development
 
+  release-approval:
+    machine: true
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run: bazel run @graknlabs_build_tools//ci:release-approval
+
+  deploy-github:
+    machine: true
+    working_directory: ~/client-java
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run: DEPLOY_GITHUB_TOKEN="$GRABL_CREDENTIAL" bazel run //:deploy-github
+
+  deploy-maven:
+    machine: true
```

Comment:
where's the test equivalent of this job, @lolski ? I.e. `deploy-maven-test`? I see that it's still bundled up in `test-deployment-maven`. I think we should split `deploy-maven-test` and it should be as clean as this `deploy-maven`, except with argument `-- snapshot`

Reply:
good point

Reply:
fixed